src/install/dracut-install.c

```diff
@@ -2079,6 +2138,47 @@ int main(int argc, char **argv)
                 }
         }
 
+        if (arg_hash_source) {
+                r = asprintf(&libdir, "%s/usr/lib/dracut", destrootdir);
+                if (r < 0) {
+                        log_error("Out of memory!");
+                        r = EXIT_FAILURE;
+                        goto finish;
+                }
+
+                r = faccessat(AT_FDCWD, libdir, F_OK, AT_SYMLINK_NOFOLLOW);
+                if (r != 0) {
+                        if (errno != ENOENT) {
+                                log_error("ERROR: lstat '%s': %m", libdir);
+                                r = EXIT_FAILURE;
+                                goto finish;
+                        } else {
+                                log_info("mkdir '%s'", libdir);
+                                r = dracut_mkdir(libdir);
+                                if (r != 0) {
+                                        r = EXIT_FAILURE;
+                                        goto finish;
+
+                                hashmap_put(items, i, i);
```

Comment:
`i`? Do you mean `libdir`?

Reply:
Actually it should not contain `destrootdir`. Must add the *file* path to the hashmap too, below, after successful `fopen()`.